src/librustc/ty/query/mod.rs

```diff
@@ -350,8 +350,9 @@ define_queries! { <'tcx>
         [] fn check_match: CheckMatch(DefId)
             -> Result<(), ErrorReported>,
 
-        /// Performs the privacy check and computes "access levels".
+        /// Performs part of the privacy check and computes "access levels".
         [] fn privacy_access_levels: PrivacyAccessLevels(CrateNum) -> Lrc<AccessLevels>,
+        [] fn check_privacy: CheckPrivacy(CrateNum) -> (),
```

Comment:
Can you rename this to `additional_privacy_checking`? It seems like most of the actual privacy checking is done in `check_mod_privacy` so I think it would be confusing if a query called `check_privacy` did not actually do everything.

Reply:
I renamed it to `check_private_in_public` instead.